src/mem/allocstats.h

```diff
@@ -183,6 +183,7 @@ namespace snmalloc
       UNUSED(sc);
 
 #ifdef USE_SNMALLOC_STATS
+      assert(sizeclass[sc].count.current > 0);
```

Comment:
Can't `current` become zero?  Should we make the statistics use a signed representation, and check for below 0?  Or perhaps, we could add an assert into `dec` that it is above 0?

Reply:
This check is performed before the decrement, and so still permits `current` to become zero.  But yes, I agree that inside `dec()` is probably clearer; will move.